src/main/java/dinkplugin/notifiers/LevelNotifier.java

```diff
@@ -104,6 +104,7 @@ private void attemptNotify() {
         createMessage(DinkPluginConfig::levelSendImage, NotificationBody.builder()
             .content(fullNotification)
             .extra(new LevelNotificationData(lSkills, currentLevels))
+            .screenshotFile("levelUpImage.png")
```

Comment:
Should probably be `levelImage.png` to be consistent with other image names in that they don't include the verb "lootReceived" or "achievmentDiaryCompleted"

Reply:
`levelImage` was my original suggestion but it read weird to me so I mentioned using something better